Makefile

```diff
@@ -47,36 +47,46 @@ TFORMAT := short
 
 export PATH:=$(GOBIN):$(PATH)
 
-env:  ## Print Go env variables.
+env: # @HELP Print Go env variables.
+env:
 	go env
 
-deps: $(GOMAJOR)  ## List available module dependency updates.
+deps: # @HELP List available module dependency updates.
+deps: $(GOMAJOR)
 	gomajor list
 
-tparse: $(TPARSE)  ## Run all tests and outputs a coverage report using tparse.
+tparse: # @HELP Run all tests and outputs a coverage report using tparse.
+tparse: $(TPARSE)
 	@go test -count=1 -json -cover $(TEST_PACKAGES) | tparse -follow -all -notests
 
-test: $(GOTESTSUM)  ## Run all tests and outputs a summary using gotestsum.
+test: # @HELP Run all tests and outputs a summary using gotestsum.
+test: $(GOTESTSUM)
 	@gotestsum --format=$(TFORMAT) $(TEST_PACKAGES)
 
-test-race: $(GOTESTSUM)  ## Run all tests with the race detector.
+test-race: # @HELP Run all tests with the race detector.
+test-race: $(GOTESTSUM)
 	@gotestsum --format=$(TFORMAT) $(TEST_PACKAGES) -- -race
 
-list-tested-packages:  ## Print a list of packages being tested.
+list-tested-packages: # @HELP Print a list of packages being tested.
+list-tested-packages:
 	$(foreach PACKAGE,$(TEST_PACKAGES),@echo $(PACKAGE)$(NEWLINE))
 
-list-ignored-packages:  ## Print a list of packages ignored in testing.
+list-ignored-packages: # @HELP Print a list of packages ignored in testing.
+list-ignored-packages:
 	$(foreach PACKAGE,$(TEST_IGNORED_PACKAGES),@echo $(PACKAGE)$(NEWLINE))
 
+lint: # @HELP Lint the project Go files with golangci-lint.
 lint: OUT_FORMAT ?= colored-line-number
 lint: LINT_FLAGS ?= --timeout=5m --fix
-lint: $(GOLANGCI_LINT)  ## Lint the project Go files with golangci-lint.
+lint: $(GOLANGCI_LINT)
 	golangci-lint run --out-format $(OUT_FORMAT) $(LINT_FLAGS)
 
-gen-goa: $(GOA)  ## Generate Goa assets.
+gen-goa: # @HELP Generate Goa assets.
+gen-goa: $(GOA)
 	goa gen github.com/artefactual-sdps/enduro/internal/api/design -o internal/api
 
-gen-dashboard-client:  ## Generate Dashboard web client from OpenAPI spec.
+gen-dashboard-client: # @HELP Generate Dashboard web client from OpenAPI spec.
```

Comment:
Nitpick: Missing articles: `@HELP Generate the Dashboard web client from the OpenAPI spec.`